exchange.py

```diff
@@ -132,78 +151,65 @@ def _getmyopenorders_tux(self):
             try:
                 if DEBUG: print("--> Getting open orders...")
 
-                query = { "method": "getmyopenorders", "nonce": int(time.time()*1000) }
-
-                post = urllib.parse.urlencode(query)
-                signature = hmac.new(self.privatekey.encode('utf-8'), post.encode('utf-8'), hashlib.sha512).hexdigest()
-                header = {'Key' : self.publickey, 'Sign' : signature}
-                response = requests.post(tuxURL, data=query, headers=header).json()
-
+                query = { "method": "getmyopenorders"}                
+                response = self._signed_request(query)
+                self.debug(3,getmyopenorders_tux,response)
```

Comment:
(Nitpick) add spaces between parameters.

Reply:
I use spaces between parameters in my code, so that'll be in the style guide (if it isn't already a PEP standard)